srcpkgs/gimx/template

```diff
@@ -0,0 +1,80 @@
+# Template file for 'gimx'
+pkgname=gimx
+version=8.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gettext pkg-config"
+makedepends="libbluetooth-devel libcurl-devel libusb-devel libX11-devel libXi-devel
+ libxml2-devel mhash-devel ncurses-devel wxWidgets-gtk3-devel"
+depends="gimx-cli-${version}_${revision} gimx-firmware-${version}_${revision}
+ avrdude xdg-utils xterm"
+short_desc="Use a computer as a hub for your gaming devices"
+maintainer="amak <amak.git@outlook.com>"
+license="GPL-3.0-only"
+homepage="https://blog.gimx.fr/"
+distfiles="https://github.com/matlo/GIMX/releases/download/v${version}/gimx_${version}-1.tar.gz"
+checksum=783673435bd424ba49e6ad5911559a517f746e50dad0f10689f0dc53336a8e1e
+patch_args="-Np1"
+
+# silence unused function errors in loader/gimx-loader.cpp
+# that are triggered by archs not defined in info.h
+CPPFLAGS+=" -Wno-unused-function"
```

Comment:
Did this work? `CPPFLAGS` are for the preprocessor, not for the compilers themselves.

Also, you should just set it, not add to it.

Reply:
That should be `CXXFLAGS` but it does work with `CPPFLAGS`.